src/components/common/payout-amounts.vue

```diff
@@ -41,6 +41,17 @@ export default defineComponent({
       } else {
         return true
       }
+    },
+
+    settingsMultiplier(label) {
+      switch (label) {
+        case 'Cash Token':
```

Comment:
@evgenibir where do this values come from? We should have it as map of token type options. and then use `TOKEN_TYPE.CASH_TOKEN` instead of `Cash Token` anywhere in code where we would like to use `Cash Token`

Reply:
@Alex-IS fixed!